Include/pystate.h

```diff
@@ -75,18 +75,39 @@ typedef struct {
     wchar_t *dll_path;      /* Windows DLL path */
 #endif
 
+    /* If greater than 0, enable isolated mode sys.path contains
+       neither the script's directory nor the user's site-packages directory.
+       Ignored if set to -1.
+
+       Related to Py_IsolatedFlag global configuration variable and -I command
+       line option. */
+    int isolated;
+
+    /* If equal to zero, disable the import of the module site and the
+       site-dependent manipulations of sys.path that it entails. Also disable
+       these manipulations if site is explicitly imported later (call
+       site.main() if you want them to be triggered).
+       Ignored if set to -1.
+
+       Related to Py_NoSiteFlag global configuration variable and -S command
+       line option. */
```

Comment:
Maybe "Initialized from Py_NoSiteFlag global configuration variable and set to 0 by -S command line option"?

Reply:
I updated the doc.